pkg/installation/install.go

```diff
@@ -72,11 +76,19 @@ func Install(p environment.KrewPaths, plugin index.Plugin, forceHEAD bool) error
 	}
 
 	glog.V(1).Infof("Finding download target for plugin %s", plugin.Name)
-	version, uri, fos, err := getDownloadTarget(plugin, forceHEAD)
+	version, uri, fos, bin, err := getDownloadTarget(plugin, forceHEAD)
 	if err != nil {
 		return err
 	}
-	return downloadAndMove(version, uri, fos, filepath.Join(p.Download, plugin.Name), filepath.Join(p.Install, plugin.Name))
+	dst, err := downloadAndMove(version, uri, fos, filepath.Join(p.Download, plugin.Name), filepath.Join(p.Install, plugin.Name))
```

Comment:
this next 3 calls (downloadAndMove, checkSubPath,createOrUpdateLink) are duplicated between here and upgrade.go with this PR.

can we move them to a common place (like an unexported method) and call them?

when it's just downloadAndMove it's cool, but when it's a 3-method sequence, avoiding repetition would be better.